### ISSUE TITLE
Conflict with Modernizr

### ISSUE THREAD

**Issue by andrey-hohlov** [stylelint/stylelint#2352]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

When installed Modernizr and gulp-stylelint lint task throw error.

```
"devDependencies": {
  "gulp": "^3.9.1",
  "gulp-stylelint": "^3.8.0"
},
"dependencies": {
  "modernizr": "^3.3.1"
}
```
1. Run `npm i` for install dependencies
2. Run `gulp lint-css`

No errors if install gulp-stylelint first, then install Modernizr:

1. Run `npm i gulp gulp-stylelint modernizr` instead of `npm i`
2. Run `gulp lint-css`

Issue in gulp-stylelint: https://github.com/olegskl/gulp-stylelint/issues/71

> Which rule, if any, is this issue related to?

irrelevant

> What CSS is needed to reproduce this issue?

irrelevant

> What stylelint configuration is needed to reproduce this issue?

irrelevant

> Which version of stylelint are you using?

gulp-stylelint `3.8.0` (stylelint `7.8.0`)

> How are you running stylelint: CLI, PostCSS plugin, Node API?

``` javascript
const gulp = require('gulp');

gulp.task('lint-css', function lintCssTask() {
  const gulpStylelint = require('gulp-stylelint');
  return gulp
    .src('style.css')
    .pipe(gulpStylelint({
    reporters: [
      {formatter: 'string', console: true}
    ]
  }));
});
```
> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

irrelevant

> What did you expect to happen?

css linted

> What actually happened (e.g. what warnings or errors you are getting)?

```
TypeError: Cannot read property '0' of undefined
    at _.omitBy (/Users/andrey/Work/Repositories/test/node_modules/stylelint/lib/formatters/jsonFormatter.js:8:48)
    at /Users/andrey/Work/Repositories/test/node_modules/lodash/lodash.js:11233:17
    at /Users/andrey/Work/Repositories/test/node_modules/lodash/lodash.js:3085:13
    at /Users/andrey/Work/Repositories/test/node_modules/lodash/lodash.js:3925:15
    at baseForIn (/Users/andrey/Work/Repositories/test/node_modules/lodash/lodash.js:2509:40)
    at basePickBy (/Users/andrey/Work/Repositories/test/node_modules/lodash/lodash.js:3084:7)
    at Function.omitBy (/Users/andrey/Work/Repositories/test/node_modules/lodash/lodash.js:11232:14)
    at results.map.result (/Users/andrey/Work/Repositories/test/node_modules/stylelint/lib/formatters/jsonFormatter.js:8:14)
    at Array.map (native)
    at module.exports (/Users/andrey/Work/Repositories/test/node_modules/stylelint/lib/formatters/jsonFormatter.js:7:34)
```

**Comment by hudochenkov** [comment]
Thank you for reporting and using issue template!

The problem is we have specified lodash as `"lodash": "^4.0.0"`, but clearly we need at least `lodash@4.0.1` or `lodash@4.5.1`. Lodash team [made changes](https://github.com/lodash/lodash/wiki/Changelog#v401) to `_.omitBy` in these versions.

Modernizr has strict dependency declaration `"lodash": "4.0.0"`. As the result, with `npm i` Modernizer get installed first with `lodash@4.0.0` which is “fine” by our `package.json`.

**Comment by davidtheclark** [comment]
Closed by #2353.
